### PR TITLE
Update to version 6.2.2

### PR DESCRIPTION
**Pull request by X-m7**
Also updated runtime to 23.08.

The release date in the appdata.xml for 6.2.2 was based on the dates I found on several sites, which all agree on 21 Nov 2023 (for example https://community.chocolatey.org/packages/geekbench and https://www.techspot.com/downloads/3943-geekbench.html).

### CODE REVIEW

**Comment by X-m7** [comment]
Seems like the AppStream validator is complaining about the screenshot now:

```
com.geekbench.Geekbench6.appdata.xml: FAILED:
• attribute-invalid     : <screenshot> height too large [https://raw.githubusercontent.com/razzeee/com.geekbench.Geekbench6/f378d2e1432c5fece2ec9ef73b631c31f0328e8f/screenshot1.png] maximum is 900px
• style-invalid         : <image> has vertical padding [https://raw.githubusercontent.com/razzeee/com.geekbench.Geekbench6/f378d2e1432c5fece2ec9ef73b631c31f0328e8f/screenshot1.png]
• style-invalid         : <image> has horizontal padding [https://raw.githubusercontent.com/razzeee/com.geekbench.Geekbench6/f378d2e1432c5fece2ec9ef73b631c31f0328e8f/screenshot1.png]
Validation of files failed
```

**Comment by X-m7** [comment]
bot, build

File: com.geekbench.Geekbench6.appdata.xml [hunk 1]
```diff
@@ -20,13 +21,18 @@
     <screenshots>
         <screenshot type="default">
             <caption>Running the vulkan benchmark</caption>
-            <image type="source">https://raw.githubusercontent.com/razzeee/com.geekbench.Geekbench6/f378d2e1432c5fece2ec9ef73b631c31f0328e8f/screenshot1.png</image>
+            <image type="source">https://raw.githubusercontent.com/X-m7/com.geekbench.Geekbench6/b38b8a195a94abc821c2e18a0decb49cd25329fa/screenshot1.png</image>
```
Comment (razzeee) [inline com.geekbench.Geekbench6.appdata.xml]: Screenshot should be made with default window controls and with shadows on the window

Comment (X-m7) [reply com.geekbench.Geekbench6.appdata.xml]: Oh, so the shadows has nothing to do with the padding errors from earlier then, no complaints from me as long as the build works lol.

Comment (razzeee) [reply com.geekbench.Geekbench6.appdata.xml]: We're in the middle of a translation on the side of flathub, so that might have been a hickup

**Comment by razzeee** [comment]
New release seems to be missing the icon, need to check if that's still in the archive

**Pull request merged**